sdk/core/Azure.Core.Experimental/src/SchemaValidator/LruCache.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections;
+using System.Collections.Generic;
+
+#nullable enable
+
+namespace Azure.Core.Experimental
```

Comment:
Stuff in Core.Experimental normally has the `Azure` or `Azure.Core` namespace, doesn't it, @JoshLove-msft?

Reply:
Yeah it looks like it does, I'll update

Reply:
Correct, the idea is to make it as close as possible to what we would eventually ship out of core so it is a simple file move.